portal_pages/models.py

```diff
@@ -259,7 +273,8 @@ def marketplace_index(self):
     MultiFieldPanel(ContactFields.panels, "Contact"),
     InlinePanel(MarketplaceEntryPage, 'services', label="Services"),
     InlinePanel(MarketplaceEntryPage, 'expertise_tags', label="Expertise"),
-    InlinePanel(MarketplaceEntryPage, 'countries', label="Locations of Expertise"),
+    InlinePanel(MarketplaceEntryPage, 'regions', label="Regions of Expertise"),
```

Comment:
"Regions of Experience" is what I recall from the meeting and what we have in notes.